lib/matplotlib/tests/test_axes.py

```diff
@@ -2250,7 +2250,7 @@ def test_contour_hatching():
     x, y, z = contour_dat()
     fig, ax = plt.subplots()
     ax.contourf(x, y, z, 7, hatches=['/', '\\', '//', '-'],
-                cmap=plt.get_cmap('gray'),
+                cmap=mpl.colormaps['gray'],
```

Comment:
Can we just use strings if we aren't using `resample` or using the `Colormap` for anything?

Reply:
It can, but I was just mechanically changing (a lot of this was done with search-and-relpace)

Reply:
I've decided to leave this.  It makes the test a bit more complicated, but I do not know if we are definitely exercising the ability to pass `Colormap` objects through this top level API else where.